### PR TITLE
Fixes from INSPIRE Labs

### PR DESCRIPTION
**Pull request by jalavik**
Here is a collage of simple one-liners that would be nice to include in the upcoming release.

They are tested on INSPIRE QA and Labs.

CC @jirikuncar

### CODE REVIEW

File: invenio/ext/arxiv/__init__.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def search(self, arxiv=None):
 
         # query the database
         result = get_unique_record_json(
-            self.app.config.get("ARXIV_SEARCH_PREFIX", "") + arxiv)
+            "\"{}\"".format(self.app.config.get("ARXIV_SEARCH_PREFIX", "") + arxiv))
```
Comment (jirikuncar) [inline invenio/ext/arxiv/__init__.py]: `"\"{}{}\"".format(self.app.config.get("ARXIV_SEARCH_PREFIX", ""), arxiv))` ?

**Comment by jirikuncar** [comment]
@jalavik isn't more for maint-2.0?

**Comment by jalavik** [comment]
Sure I can rebase there

**Comment by jalavik** [comment]
Done. Note that to be compatible with maint-2.0 I removed the `"` around the table name in the BibIndex fix. When merging you'll have to make sure it's added back.

**Pull request merged**